converters/fromZigbee.js

```diff
@@ -1931,7 +1931,57 @@ const converters = {
             return {pressure: precisionRoundOptions(pressure, options, 'pressure')};
         },
     },
-
+    AC0251100NJ_on: {
+        cid: 'genOnOff',
+        type: 'cmdOn',
+        convert: (model, msg, publish, options) => {
+            return {action: 'up'};
+        },
+    },
+    AC0251100NJ_off: {
+        cid: 'genOnOff',
+        type: 'cmdOff',
+        convert: (model, msg, publish, options) => {
+            return {action: 'down'};
+        },
+    },
+    AC0251100NJ_on_hold: {
+        cid: 'genLevelCtrl',
+        type: 'cmdMoveWithOnOff',
+        convert: (model, msg, publish, options) => {
+            return {action: 'on_hold'};
+        },
+    },
+    AC0251100NJ_off_hold: {
+        cid: 'genLevelCtrl',
+        type: 'cmdMove',
+        convert: (model, msg, publish, options) => {
+            return {action: 'off_hold'};
+        },
+    },
+    AC0251100NJ_release: {
+        cid: 'genLevelCtrl',
+        type: 'cmdMoveToLevelWithOnOff',
+        convert: (model, msg, publish, options) => {
+            return {action: 'circle_press'};
```

Comment:
What is the difference with the circle_click here? (do you e.g. have to press the button longer?)

Reply:
you have 3 stats here.. pressed, released, hold